audio/src/audio_vlc.py

```diff
@@ -0,0 +1,139 @@
+import os
+import time
+import threading
+os.add_dll_directory(r"C:\Program Files (x86)\VideoLAN\VLC")
```

Comment:
This looks platform-specific, what will happen on Linux?

Reply:
Oh true, I had that as a ToDo but forgot to fix. My plan is to add multiple paths so they cover all possible OSes we have, does that make sense?

Reply:
I'm not sure how it would look in practice, but as long as it's handled I think that's fine.